### ISSUE TITLE
Windows development not working (with the CheckNetIsolation command)

### ISSUE THREAD

**Issue by chris-pikul** [neutralinojs/neutralinojs#508]
**Expected Behavior**  
Development on Windows should show the window when using either `neu run` or `neu listen`.
  
**Actual Behavior**  
Blank window with no title, no icon, no content. 

Running in "browser" mode does work. So does building with and without the --release flag. Only local development seems to be effected.

**Steps to Reproduce the Problem**  
 1. `neu create example`
 2. `CheckNetIsolation.exe LoopbackExempt -a -n="Microsoft.Win32WebViewHost_cw5n1h2txyewy"`
 3. `cd example` then `neu run`

**Specifications**  
Neutralinojs binaries: v2.4.1
Neutralinojs client: v1.0.0
neu CLI: v5.1.5
Windows 10 Home version 21H1

![image](https://user-images.githubusercontent.com/4327937/124676497-8346e980-de73-11eb-9063-f435f2d4a989.png)

**Comment by AndrewvanDijk** [comment]
Seriously, such promise, and yet such a waist of time, fix this problem. Whats the point of your project of the application doesnt work.

**Comment by rince1984** [comment]
@chris-pikul but you executed the cmd command in admin mode (right click on cmd.exe > Run as administrator)?

**Comment by chris-pikul** [comment]
> @chris-pikul but you executed the cmd command in admin mode (right click on cmd.exe > Run as administrator)?

Yes, using powershell, the command executed properly.

**Comment by mzy2240** [comment]
Facing same problem. BTW, how to enable browser mode?

**Comment by rince1984** [comment]
> Facing same problem. BTW, how to enable browser mode?

Just edit the config file (neutralino.config.json). In the second line (defaultMode) you can set "browser" to start the app in browser mode.

**Comment by mzy2240** [comment]
Thanks! Actually I found the option just after comment, but cannot find this issue somehow lol. I tried all the configurations I could find online that work with Vue, but neither work. For the same dist, I am able to build Electron and Tauri without issues. No errors so really hard to debug.

**Comment by rince1984** [comment]
I don't think the white screen problem has anything to do with using Vue. This is usually a Windows issue regarding the AWP app's permission to communicate with localhost. So far I have been able to fix the white screen on every Windows 10 machine by using the appropriate CMD command.

For me the command ```CheckNetIsolation.exe LoopbackExempt -a -n="Microsoft.Win32WebViewHost_cw5n1h2txyewy"``` works if I open Command Line with Administrator rights.

Electron and Tauri are not based on AWP, I think.

**Comment by mzy2240** [comment]
Yeah I also don't think the issue I am facing has anything to do with the AWP permission. When using window mode, nothing in the index.html; when using browser mode, a json shows up with hotplug info. Using pure vue-cli to serve the website works fine, so really don't know where is the problem. Tried changing the URL but no luck.

**Comment by shalithasuranga** [comment]
Closing this since this can be solved by installing the WebView2 runtime :tada: Thanks everyone
